src/perotf_s/__init__.py

```diff
@@ -390,6 +391,47 @@ class peroTF_UP_SlotDieBox_SlotDieCoating(SlotDieCoating, EntryData):
         a_template=dict(
             layer_type="Absorber Layer"))
 
+# # %% ### Annealing
+
+class peroTF_CR_ThermalAnnealing(Annealing, EntryData):
```

Comment:
this would need to be inherriting from BaseProcess. like this:
```python
class peroTF_CR_ThermalAnnealing(BaseProcess, EntryData):
....
    annealing = SubSection(
        links=['http://purl.obolibrary.org/obo/RO_0001019'],
        section_def=Annealing)
```